packages/paste-website/src/pages/inclusive-design/index.mdx

```diff
@@ -0,0 +1,409 @@
+      </UnorderedList>
+    </DisclosureContent>
+  </Disclosure>
+  
+  <Disclosure>
+    <DisclosureHeading as="h3" variant="heading30">
+      Assistive technology considerations
+    </DisclosureHeading>
+    <DisclosureContent>
+      <UnorderedList>
+        <ListItem>
+          Use screen reader and keyboard accessible HTML{' '}
+          <Anchor showExternal href="https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML">(Read more about why this matters)</Anchor>
+        </ListItem>
+        <ListItem>Honor requests to remove animation via the `prefers-reduced-motion` media query</ListItem>
+        <ListItem>Do not recreate supported and expected browser behaviors with bespoke scripts</ListItem>
+        <ListItem>
+          Support Windows high contrast mode (use images, not background images){' '}
+          <Anchor showExternal href="http://adrianroselli.com/2012/08/css-background-images-high-contrast-mode.html">
+            (Read more about why this matters)
+          </Anchor>
+        </ListItem>
+        <ListItem>
+          Apply `alt="` or `aria-hidden="true"` to decorative images{' '}
+          <Anchor showExternal href="https://www.w3.org/WAI/tutorials/images/decorative/">(Read more about why this matters)</Anchor>
+        </ListItem>
+        <ListItem>Use data tables (`table`) for data only, not visual layout purposes</ListItem>
+        <ListItem>Make scrollable elements focusable for keyboard users</ListItem>
+        <ListItem>Ensure keyboard focus order is logical regarding visual layout</ListItem>
+        <ListItem>Do not hijack standard scrolling behavior</ListItem>
+        <ListItem>Move focus between dialogs and the controls that invoked them</ListItem>
+        <ListItem>Provide status and error messages as WAI-ARIA live regions</ListItem>
+        <ListItem>Provide clear, unambiguous focus styles</ListItem>
+        <ListItem>Ensure states (pressed, expanded, invalid, etc) are communicated to assistive software</ListItem>
+        <ListItem>Match semantics to behavior for assistive technology users</ListItem>
+        <ListItem>Ensure disabled controls are not focusable</ListItem>
+        <ListItem>
+          Provide a skip link if necessary{' '}
+          <Anchor showExternal href="https://webaim.org/techniques/skipnav/">(Read more about why this matters)</Anchor>
+        </ListItem>        
+        <ListItem>Make sure controls within hidden content are not focusable</ListItem>
+      </UnorderedList>
+    </DisclosureContent>
+  </Disclosure>
+  <Disclosure>
+    <DisclosureHeading as="h3" variant="heading30">
+      Other considerations
+    </DisclosureHeading>
+    <DisclosureContent>
+      <UnorderedList>
+        <ListItem>
+          Make sure main body (paragraph) text is no smaller than the default (user agent) size{' '}
+          <Anchor showExternal href="https://www.smashingmagazine.com/2011/10/16-pixels-body-copy-anything-less-costly-mistake/">
```

Comment:
We don't do this, and I don't expect we ever will because of data density issues. Should we remove it?

Reply:
:( ok